conjur-openapi.yml

```diff
@@ -25,6 +25,8 @@ tags:
   description: "SSH keys"
 - name: "resources"
   description: "Resources"
+- name: "authenticators"
+  description: "Authenticators"
```

Comment:
is this the right name? we already have 
```
- name: "authn"
  description: "Authentication"
```

maybe @alexkalish has thoughts?

Reply:
I'm wondering in particular if the `get '/authenticators' => 'authenticate#index'` route should just be included under the `authn` section

Reply:
I originally did group them together. However because the /authenticators endpoint doesn't actually perform any authentication it didn't seem quite right. I think it should be included in the `status` tag created for the /whoami endpoint. It retrieves the "status" of the authenticators on the server so I think it fits a little more nicely.